SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/HomeViewController.swift

```diff
@@ -166,12 +170,26 @@ extension HomeViewController: UICollectionViewDelegateFlowLayout {
 }
 
 extension HomeViewController: SliderListDelegate{
+    func hideSectionsWithoutData() {
+        if self.sliderViewModel?.getNewsCount() == 0 {
```

Comment:
the VC should not decide when to hide the data, the code it's nice but the if should not be here